### ISSUE TITLE
auth_url_return param not working if already logged in (continuation)

### ISSUE THREAD

**Issue by jaitor1** [3liz/lizmap-web-client#1980]
Related to issue: https://github.com/3liz/lizmap-web-client/issues/1957

Thanks for your report

_Originally posted by @laurentj in https://github.com/3liz/lizmap-web-client/issues/1957#issuecomment-739961703_
------------------------------

I tried the changes and it is not working as expected, at least to me.

Now I am not redirected to "USER, You are connected." site but I am redirected to the Admin Desktop:

![image](https://user-images.githubusercontent.com/12136796/101770969-877f7400-3ae9-11eb-9e04-e4cd34b68cb3.png)

I should be redirected to the auth_url_return URL. Maybe I am doing something wrong or I have a misconfiguration somewhere. My URL example:

http://MYSERVER/lizmap/www/admin.php/auth/login?auth_url_return=%2Flizmap%2Fwww%2Findex.php%2Fview%2Fmap%2F%3Frepository%3Druidos%26project%3Dmapa_ruido

**Comment by jaitor1** [comment]
I am still experiencing this issue. My environment is the same as detailed in the original issue  #1957

You can try the issue in this link:

[http://eudala2.getxo.eus/lizmap/www/admin.php/auth/login/in?&auth_login=borrar&auth_password=borrar_test&auth_url_return=%2Flizmap%2Fwww%2Findex.php%2Fview%2Fmap%2F%3Frepository%3Dtest%26project%3Dtest](url)

I created this test map and dumb user/pass for this test. I modified the login form and added the allowAnyOrigin="true" to be able to log in via url parms. Don't worry about security, it is only used within a local network, in a controlled environment.

1. First time you click on the link you get redirected to the login website. It does not log in although I have already entered the user and pass via URL params and it does not redirect to the specified map neither.

2. But second time you click on the link you get logged in correctly and redirected to the map properly.

3. Third time + you are already logged in correctly but you get always redirected to the admin website instead of to the map. 

In short, it only works the second time you hit the link.

**Comment by Gustry** [comment]
In QGIS desktop, Lizmap plugin, please right click on your server and copy/paste versions here

**Comment by jaitor1** [comment]
> In QGIS desktop, Lizmap plugin, please right click on your server and copy/paste versions here

My plugin is not getting versions correctly.

![image](https://user-images.githubusercontent.com/12136796/182837969-896bec5c-9fa2-44dd-a825-5a25fe34018b.png)


The environment is:

Lizmap version: 3.3.3
QGIS Server FULL version: 3.10.10 LTR
QGIS Desktop FULL version: 3.10.10 LTR
OS (Windows, Linux, MacOS, Android…): Windows Server 2016 Standard.
Lizmap admin information (users, logs…)? Everything default

**Comment by jaitor1** [comment]
I will open a new issue as this one is closed and may not have the attention it requires.
